packages/components/src/Enumeration/Items/Items.scss

```diff
@@ -8,6 +8,7 @@ $tc-enumeration-height: 40vh !default;
 	list-style: none;
 	box-shadow: inset 0 21px 3px -20px $shadow, inset 0 -21px 3px -20px $shadow;
 	height: $tc-enumeration-height;
+	max-height: $tc-enumeration-height;
```

Comment:
Since you set the max-height, is the height still relevant ?

Reply:
If I remove the height it will be a braking for the existing component.
And in the case of dynamic height, we have the max height to avoid unlimited list.